rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java

```diff
@@ -49,22 +53,23 @@
 
     private final JobKeyValueTransformer jobKeyValueTransformer;
 
-    public SubmitJobCommand(String... params) {
-
+    public SubmitJobCommand(String... params) throws CLIException {
+        if (params == null || params.length == 0) {
```

Comment:
We need also to check if the object "params" is not null. If we test only on the length, we will get NullPointerException if "params" is null.